projectroles/tests/test_commands.py

```diff
@@ -602,9 +604,19 @@ def test_command_cleanappsettings(self):
                     (
                         CLEAN_LOG_PREFIX
                         + 'Deleting "settings.example_project_app.ghost" '
-                        'from project "{}"'.format(self.project.title)
+                        'from project "{}" - definition not found'.format(
+                            self.project.title
+                        )
+                    ),
+                    # Check for project type
+                    (
+                        CLEAN_LOG_PREFIX
+                        + 'Deleting "settings.example_project_app.project_int_setting" '
+                        'from project "{}" - project type "{}" does not match allowed types: {}'.format(
```

Comment:
I recommend defining these error strigns as constants in the command module, importing the constants to tests and checking against those instead. Asserting long strings can very easily become annoying when we e.g. want to change the test in code and have to go to tests to modify the strings there.